source/ec_override.c

```diff
@@ -13,22 +13,7 @@
  * permissions and limitations under the License.
  */
 
-#include <limits.h>
-
-#include <bn_utils.h>
-#include <ec_utils.h>
 #include <openssl/ec.h>
```

Comment:
I think we could `#include <ec_utils.h>` here and remove `#include <ec_utils.h>` from `openssl/ec.h` as I mentioned in a previous comment.
That way we could be consistent with the style we use for `bn.h` and `bn_utils.h`.

Reply:
Implemented as suggested.